scio-extra/src/main/scala/com/spotify/scio/extra/sparkey/instances/MockSparkeyReader.scala

```diff
@@ -74,7 +75,10 @@ case class MockByteArrayEntry(k: Array[Byte], v: Array[Byte])
 
 case class MockByteArraySparkeyReader(data: Map[Array[Byte], Array[Byte]])
     extends MockSparkeyReader {
-  override def getAsByteArray(key: Array[Byte]): Array[Byte] = data(key)
+  private lazy val internal = data.map { case (k, v) => (ByteBuffer.wrap(k), v) }
```

Comment:
Using ByteString might be better here.

Reply:
true, although that bring in a protobuf dependency to scio-sparkey where there was none before

probably not a big deal

Reply:
why not using scala `Seq[Byte]` or `Vector[Byte]` ?

Reply:
good idea! updated